### PR TITLE
Create multinode env

### PR DESCRIPTION
**Pull request by jackhodgkiss**
Create `kayobe-config` environment called `ci-multinode`. These three commits represent the initial configuration of the environment. Majority of the content is copied from `ci-aio`. You can find the actual changes within;

- `etc/kayobe/environments/ci-multinode/inventory/hosts`
- `etc/kayobe/environments/ci-multinode/network-allocation.yml`
- `etc/kayobe/environments/ci-multinode/networks.yml`

### CODE REVIEW

File: etc/kayobe/environments/ci-multinode/inventory/hosts [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# A single all-in-one controller/compute host.
```
Comment (markgoddard) [inline etc/kayobe/environments/ci-multinode/inventory/hosts]: This file should be generated by terraform

File: etc/kayobe/environments/ci-multinode/network-allocation.yml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+admin_ips:
```
Comment (markgoddard) [inline etc/kayobe/environments/ci-multinode/network-allocation.yml]: This file should be generated by terraform. Although I thought we also needed some other config for the admin network - CIDR, gateway, etc.

**Review by markgoddard — CHANGES_REQUESTED** [review]

File: etc/kayobe/environments/ci-multinode/network-allocation.yml (hunk 2 above)
Comment (jackhodgkiss) [reply etc/kayobe/environments/ci-multinode/network-allocation.yml]: The admin network config is included in `tf-networks.yml` which will be included in the next PR.

Comment (markgoddard) [reply etc/kayobe/environments/ci-multinode/network-allocation.yml]: Shouldn't that be generated by TF?

**Review by markgoddard — APPROVED** [review]

**Pull request merged**